static/js/reducers/ui.js

```diff
@@ -25,8 +26,24 @@ import {
 } from '../actions/ui';
 import { HIGH_SCHOOL, ASSOCIATE, BACHELORS, MASTERS, DOCTORATE } from '../constants';
 import { calculateDegreeInclusions } from '../util/util';
+import type { Action } from '../flow/generalTypes';
 
-export const INITIAL_UI_STATE = {
+export type UIState = {
```

Comment:
Would be nice to combine this and `INITIAL_UI_STATE` below somehow, not sure if that's something flow supports right now though